src/core/OmniFormElement.ts

```diff
@@ -115,6 +122,31 @@ export class OmniFormElement extends OmniElement {
      */
     @property({ type: Boolean, reflect: true }) disabled = false;
 
+    /**
+     * Toggles the ability to clear the value of the component.
+     * @attr
+     */
+    @property({ type: Boolean, reflect: true }) clearable = false;
+
+    protected _onClearClick(e: MouseEvent) {
```

Comment:
Should this not be named to indicate the action being performed. e.g. `clearValue`

Reply:
Updated to name to clearValue